README.md

```diff
@@ -1 +1,64 @@
-# To-Do-list
\ No newline at end of file
+# To-Do-list
+
+## Built With
+
+- HTML
+- CSS
+- js ES6
+- webpack
+
+## Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+## Live Demo 
+
+comming soon...
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- text editor
+
+- nodejs installed on your local machine
+
+### Setup
+
+- Clone the repository
+
+- Open the project in your favourite text editor
+
+### Install
+
+git clone https://github.com/boo-shehab/To-Do-list.git
+
+
+## 👤 Author1
+
+> ahmed abas
+- GitHub: [@boo-shehab](https://github.com/githubhandle)
+- Twitter: [@boo_shehab2000](https://twitter.com/boo_shehab2000)
+- LinkedIn: [@ahmed-al-ali](https://www.linkedin.com/in/ahmed-al-ali-77b6a0246)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc...
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [OPTIONAL] The MIT/LICENSE link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Also, I suggest removing this `NOTE ...` recommendation from your README file 👍